.github/workflows/push-tag.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.6', '3.7', '3.8', '3.9' ]
```

Comment:
there's another PR that looks to cover this. should we drop the change from this one?